.goreleaser.yml

```diff
@@ -0,0 +1,45 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod download
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - id: grpcserver
+    binary: grpcserver
+    dir: cmd/server
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
```

Comment:
Linux is the only supported kube-apiserver target, so we don't need a windows or darwin build